src/AppBundle/Controller/Admin/BlogController.php

```diff
@@ -164,16 +160,15 @@ public function editAction(Post $post, Request $request)
 
         return $this->render('admin/blog/edit.html.twig', [
             'post' => $post,
-            'edit_form' => $editForm->createView(),
-            'delete_form' => $deleteForm->createView(),
+            'form' => $form->createView(),
         ]);
     }
 
     /**
      * Deletes a Post entity.
      *
      * @Route("/{id}", name="admin_post_delete")
-     * @Method("DELETE")
+     * @Method("POST")
```

Comment:
if you change it to POST (which is something I do all the time to avoid having to enable the HttpMethodOverride which is disabled by default), I suggest changing the URL to ``/{id}/delete`` though